migrations/v2.js

```diff
@@ -0,0 +1,134 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getCourse, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('Resources - v2.0.1 to v2.0.2', async () => {
+  let course;
+  whereFromPlugin('Resources - from v2.0.1', { name: 'adapt-contrib-resources', version: '>=2.0.0 <2.0.2' });
+
+  whereContent('Resources - where Resources', async (content) => {
+    course = getCourse();
+    return course?._resources;
+  });
+
+  mutateContent('Resources - add _drawerOrder attribute', async (content) => {
+    course._resources._drawerOrder = 0;
+    return true;
+  });
+
+  checkContent('Resources - check _drawerOrder attribute', async (content) => {
+    const isValid = _.has(course, '_resources._drawerOrder');
+    if (!isValid) throw new Error('Resources - _drawerOrder attribute not updated');
```

Comment:
I think this should be [_isEnabled](https://github.com/adaptlearning/adapt-contrib-resources/commit/c557d3b206563d1eacffc742b8f6cd10e66b1396) since `_drawerOrder` is added in 2.1.0 below.
```suggestion
  mutateContent('Resources - add _isEnabled attribute', async (content) => {
    course._resources._isEnabled = true;
    return true;
  });

  checkContent('Resources - check _isEnabled attribute', async (content) => {
    const isValid = _.has(course, '_resources._isEnabled');
    if (!isValid) throw new Error('Resources - _isEnabled attribute not updated');
```